src/app/search/FilterHelp.tsx

```diff
@@ -72,11 +94,23 @@ export default function FilterHelp() {
   );
 }
 
-function FilterExplanation({ filter }: { filter: FilterDefinition }) {
+function FilterExplanation({
+  filter,
+  suggestionContext,
+}: {
+  filter:
+    | FilterDefinition<Loadout, LoadoutFilterContext, LoadoutSuggestionsContext>
+    | FilterDefinition;
+  suggestionContext: LoadoutSuggestionsContext | SuggestionsContext;
+}) {
   const dispatch = useDispatch();
-  const suggestions = Array.from(
-    new Set([...generateGroupedSuggestionsForFilter(filter, true, {})]),
+  let suggestions = Array.from(
+    new Set([...generateGroupedSuggestionsForFilter(filter, suggestionContext, true)]),
   );
+  if (filter.format === 'freeform' || filter.format?.includes('freeform')) {
+    suggestions = suggestions.slice(0, 5);
+  }
```

Comment:
Question: why limit freeform to 5 here but not other places?

Reply:
Because this is the filter help screen, and showing a list of every item name in your inventory in the filter examples seems excessive. But being able to autocomplete any item name is useful.